packages/react-redux/src/slices/cartoSlice.js

```diff
@@ -204,6 +204,8 @@ export const createCartoSlice = (initialState) => {
  * @param {FiltersLogicalOperators=} data.filtersLogicalOperator - logical operator that defines how filters for different columns are joined together.
  * @param {import('@deck.gl/carto').QueryParameters} data.queryParameters - SQL query parameters.
  * @param {string=} data.geoColumn - (optional) name of column containing geometries or spatial index data.
+ * @param {number=} data.dataResolution - data resolution for spatial index data.
```

Comment:
Is this connected to `spatialDataType` and `spatialDataColumn`, and would `spatialDataResolution` be a clearer name if so? For my own learning, is more information on what the parameters do available somewhere?

Reply:
We don't understand connection "well" enogh yet.

`dataResolution` is property of tileset - namely, maximum resolution of h3 or quadbin index encountered in table.

To apply filters, we need to apply _some_ resolution that is not bigger than `dataResolution`

The idea is that we allow users to provide two:
 * `dataResolution` - so c4r is smart enough to determine some resolution from property received in map instantiation (`dataResolution` exists in tilejson response)
 * `spatialFiltersResolution` - if user (like builder) wants to control this value directly.
 
 
That's why we have this `getHexagonResolution` - it's only used when `spatialFiltersResolution` is needed (h3/quadbin) and user didn't provided it, so we derive it from `dataResolution`.

Reply:
Ok, thanks! I guess it's a request for the API team more, but a glossary in docs would be really helpful. 😓